index.html

```diff
@@ -0,0 +1,133 @@
x1jGFjdEQ3yZ30NrkzJUxCbVNnqzIVRkchVusdZ3teoiJoV448zUVquxxXBvCW6OQyyaj1/ZDLa87ZcubaJbypirZ5DTy5AGkVWukS1OJFRQMaA9WMPVlGvr+yGzKmbkufNnO7NSbZTZWUUwUuz3weVVurCOrT7dq3Gc856Df8A6xb9F5q/nOl0eg0iUpodPf0tKUudHCw6YX5Kzps9oMpZbOXmS0mXdG5CB1LKJjqCiGgOBONIYUyETMU6aZ5y8wzltF2XQyxK83pohhS5jTaQajVDnYszklTpE1JhAkIstQEUTHVUKXZk0YulSWuka9RjpnGlIpFyHm1Hf9g95j1I+V9H7Y5P4f8AKPeYEn5X0ftEefpv/UvubT+hh9vHxh7PdEJ8qNlEyRIUvcrNdK3b+EyU6MaXl1Bifsia25vjD1D3RC/KfPKybPRglbSAXKo1Pi5jVGkBAxAjoWe+PHmyv+MilpkFESeZjLKZjbEIlhmGknS7qsDMNCNF8gDhVrrideT4UsKITUS3mylN25VUcjEXm46417BFebuTZxlpfT/tElqmiS4HEhp6mt2gBKKt39qtIsbMZT5jKmGgM0vPuqAFUTmLqoAA1KVx4zWNtamqvm8/2Uq+roP8pgLzHAAVJ2AVqYjOVJ1gcCZLmSncTHcAzDdZno9HF4ArfRGoa0u1AwiT2ZLxKnUcNVeI1w44RsbKTUyiTrqbLM2ga7vTr8IrpISlDMV+SbGs4EGSrLYkaTccGYpNw6SY9XZSjGrHfG6CuOoKB8kUdTwv5vtguWbPpJctZdGa+6VklKGUKE74AqeIGmzaKmDhV2n7Yx1kJxcd3mWqaaeBRlE4J/N74EcylqT+b3wI+gXBxjvaOA30T7oyJJtjSbQJyUvS5omLUVFUe8tRxioEa0a1KwIxxBHpikp3kbthY0mWcipoS0ytK4V3muJA1fDRlTbJ7r+8A+WjKe2T3X94cvgZtnLs3rTPw478DNs5dm9aZ+HADNYPK7lKTLEtGlXRWlZdTianGvTBXwo24rddbO4BvAPJQ4kFSek0JxMSAeRa185Z/Wf8OB8C1q5yR6z/AIcARxfKVbBdN2QWWlWMoF3AS5R2Jq4K0wOwbIOsXlZyjKrcaVQ4kGXUV6BWg7IfvgVtfOSPXf8ADgfAra+ckes/4cAQW1Z12iZa2trFNK7Bm3u9wUJS7soNUSOz+WXKaKEUyqKKD4uuA1DXDv8AApa+ckeu/wCHA+BS185I9d/w4AhOWM77TapumtDLMfGjEcFT8lFrRV6h11giz5xzkDhbovi6d7xV4scDE9+BS185I9d/w478Clr5yR6z/hwwicvGCuZWV5imq3R1LgesccdXLM0GtR6IsX4FLXzkj13/AA4SWjyRWpHuEyjhUEM1D1EpriNqJ3Mgxy1MuXBdU0oXUUcj6VeP7BDjmvntabAXNn0YLgBryXq0JI48NZiTfBLatsn1z9yB8Etq2yfXP3InBUb/AIXco7ZXd/3jk3ys5RZWUmWAwKmiUOIpUGuBxhy+CW1bZPrn7kc+CW1bZPrn7kY92q52r2L75eZD7BnLNlNKZRLOifSIGQMoe4su9TbRFavKxjs7Oaczu4WUrPM0rFUC74oyED9nfsabTWJrY/I7a5ilg0lRUjfM+NOMUTVxdhg74FLXzkj13/DjTbHPBXLIjkPPq1WUu8oSrzhAzMgLUlqqKK1rSig021MOZ8r2Udsru/7w9/Apa+ckeu/4cD4FLXzkj13/AA4pKiqTzKKJU5LhjKnlgykNRld3/eO/DHlPbJ7oeMPPwKWvnJHrv+HA+BS185I9Z/w4lUVpYUUN78xhl+Ve3qxdRIDGuIl44mpGvVXGkG/DFlPbJ7seMPPwK2vnJHrv+HHPgWtfOSPXf8ODprfghufmMr+V3KRNaytn6P8AvDnml5TLdPtkmQ5l3JkxUaiUNDroa4Qf8C1r5yR67/hwsyH5KbVZrTKtBeQVlOHajOWoNlUHviFp6ovcorP2G+WMZLWtdmZnJAwoPdCafk6+KPLVwNQYBhXbQw6mORzT0Fcpubbyy6tklgadyV5pOI8BdYFBxbMOqDUsTKAFUADAAUAAGoADUIcYItaOwCobtW3zDWFoSbteM0A6K14oq/h1T5b9x2zCZdmYEmh7DjHvRPtf1j4wZKs1w1Vn6Qzs9enfkkHqg+NIaOEFiLfuVdjfUSaN/wBv0nxgnzJ71accOECE9DCeNzfT1JVrXA25X1J/N7xAgZdlu1y4R8quFddKe4wI7DMpALnHy7R3i+Mev+Scu0euvjF+eZpsgeZpsgCg/wDknLtPeL4xz/knLtPeL4xfvmabIHmabIAoL/knLtPeL4x3/knLtPeL4xfnmabI75mmyAKCrnJy7T3i+MCucnLtPeL4xfnmabIHmabIAoOucnLtPeL4wK5y8u094vjF+eZpsgGxpsgCg65y8u094vjAvZy8u094vjF++ZpsjnmabIAoO9nLzlp7xfGBezl5y094vjF+eZpsgeZpsgChL2cvLtPeL4wK5y8u094vjF+eZpsgeZpsgCgr2cnLtPeL4wL2cnLtPeL4xfvmabIHmabIAoK9nLzlp7xfGBXOTl2nvF8YvzzNNkDzNNkAUHXOTl2nvF8YF7OTnLT3i+MX55mmyB5mmyAKDvZycu094vjArnJy7T3i+MX75mmyB5mmyAKCrnJy7T3i+McrnJy7T3i+MX75mmyO+ZpsgCga5ycu094vjHT/AKk5do7xPGL9FjTZHPM02QBEM189W0SyrdKaRPRQG1MrbGF0mnUYe1zqsp/9XV0MPshK2T5ZJJGJOMedz5eyAFv+qLLzo9Bjy+c9lp+lHoMJtzJeyObnS9kAGyM5LKCSbQzV5QwA6KKPbjClM5LMdUwHsMItzZeyPIyfL2QA5bvyOX/4nwjhzgkVoGqeogekwUtnWgw4o75uuyAHaxoGF+oa9SlNQA1AekxyC8kigZRqBw7YEAf/2Q=="
```

Comment:
I think it would be better to use jpg for this ```img``` tag 😃.

it would be more readable for code reviewers. it would be more clean code 👍

Reply:
- [ ] Good observation, as it could potentially reduce the lines occupied :+1: but to make the communication more professional, I suggest you start your sentence with an `Uppercase letter`, also after every period (`.`). This will ensure clear communication as it improves the readability and understandability of your comments.
E.g. `It would be more readable. It would be more clean code`.
- [ ] [OPTIONAL]  Consider a plus to include an example to ensure that the student grasps your suggestion without any misconstruing regarding the use of images with a `jpg`, `png` or `svg` extension like this: `<img src="image.jpg" alt="description of the image here">`. By this this, he is also shown how small/minimal the code can be compared to the previous code. How motivating! :smile: